src/live.asp.net/wwwroot/js/site.js

```diff
@@ -6,6 +6,13 @@
 
     var months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
 
+	var countdownPollInterval = 500;
+
+	var liveShowPollInterval = 15 * 1000;
+
+	var SHOW_STATUS = { "OffAir": 1, "OnAir": 2, "Standby": 3 };
+	Object.freeze(SHOW_STATUS);
```

Comment:
Yes according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze 

Feature|	Firefox (Gecko)|	Chrome|	Internet Explorer|	Opera|	Safari
------------ | ------------- | ------------- | ------------- | ------------- | -------------
|Basic support|	4.0 (2)|	6|	9	|12|	5.1|

but it could also work without it.